spec/components/button_spec.rb

```diff
@@ -141,4 +141,32 @@ def component_name
 
     assert_select ".gem-c-button--inline", text: "Button one"
   end
+
+  it "renders with a name and a value attribute  when name and value are set" do
```

Comment:
Nitpick: is that a double space between attribute and when?

Reply:
🤦‍♂